test/it/ServicesISpec.scala

```diff
@@ -38,6 +38,21 @@ class ServicesISpec extends AnyWordSpec with Matchers with GuiceOneAppPerSuite w
   private val partiallyCompliantWithoutMilestones: Seq[String] =
     Seq("pay-by-bank", "pay-by-bank.cy", "example-partially-compliant-no-milestones")
 
+  // TODO: These statements have formatting in their milestones that fail the regex for WCAG version matches text in
+  // milestones. Will need to be manually amended, we should work with DIAS on this
+  private val wcagMilestoneFormattingKnownIssues: Seq[String] =
```

Comment:
I did let DIAS know about these services on this [thread](https://hmrcdigital.slack.com/archives/C01RVNY177E/p1698402645437469) but not sure if they have addressed it or are planning to